pkg/didcomm/dispatcher/api.go

```diff
@@ -76,20 +76,19 @@ type OutboundCreator func(prov Provider) (Outbound, error)
 type DIDCommAction struct {
 	// DIDComm message
 	Message DIDCommMsg
-	// Callback function to be called by the consumer for further processing the message.
-	Callback Callback
+	// Continue function to be called by the consumer for further processing the message.
+	Continue Continue
+	// Fail invocation notifies the service that the consumer action event processing failed.
```

Comment:
comment might need some word-smithing (or that the consumer wants to stop processing).

Reply:
[Done](https://github.com/hyperledger/aries-framework-go/compare/12e97e25c20fc563333b1f7ea8f0de7c09fe1c0e..42c5b3f270b75af9cd92cb0657abbcacffce467f)